docker-app/qfieldcloud/notifs/cron.py

```diff
@@ -43,12 +44,14 @@ def do(self):
                     logging.warning(f"{user} has notifications, but no email set !")
                     continue
 
+                QFIELDCLOUD_HOST = os.environ.get("QFIELDCLOUD_HOST", None)
```

Comment:
```suggestion
                QFIELDCLOUD_HOST = os.environ.get("QFIELDCLOUD_HOST")
```
The form you used is already built-in.

Reply:
thanks

Reply:
Don't you have access to `settings.QFIELDCLOUD_HOST`? Ideally random reading from the env should be discouraged outside `settings.py`?